cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -349,13 +350,22 @@ var (
 			}
 			paths := util.UpdateFieldMask(cmd.Flags(), setEndDeviceFlags, attributesFlags())
 
+			abp, _ := cmd.Flags().GetBool("abp")
+			multicast, _ := cmd.Flags().GetBool("multicast")
```

Comment:
I still think that we should have a `abp = abp || multicast` here
Multicast implies ABP

All multicast devices are ABP
Not all ABP devices are multicast